ql/src/java/org/apache/hadoop/hive/ql/log/HushableRandomAccessFileAppender.java

```diff
@@ -51,17 +47,6 @@
 public final class HushableRandomAccessFileAppender extends
     AbstractOutputStreamAppender<RandomAccessFileManager> {
 
-  private static final LoadingCache<String, String> CLOSED_FILES =
```

Comment:
this seem to be removing the changes of HIVE-22753

I'm not against it - but since that ticket didn't added any tests; I wonder what will prevent that issue from reoccuring again?

Reply:
The `TestOperationLoggingAPIWithTez` verifies that certain messages appear in the operation logs so it is a good guard against regressions. Apart from adding more log messages to assert in `TestOperationLoggingAPIWithTez` I don't have better ideas for ensuring that it will not break in the future.

I guess HIVE-22753 didn't add any tests cause it was targeting a memory leak and these stuff are not always easy to unit test. In that case I guess it would require executing hundreds or thousands of queries till the problem shows up.

If you have ideas on how we could improve these tests, I would be happy to make any changes.